existential-vs-universal-quantification/article.md

```diff
@@ -0,0 +1,295 @@
+# Existential and Universal Quantification in Haskell
+
+In this article I give a brief introduction into one of the most common Haskell extensions – `ExplicitForAll` – and syntax constructions that this extension brings to language.
+It is used in about all big haskell projects, so it is very nice to either become acquainted with it or to refresh it in your memory if you already know about it.
+
+Stay tuned if you want to know how to create lists of differently typed variables, how to add complex constraints, and more!
+
+## Introduction
+
+By default, quantification in Haskell is implicit.
```

Comment:
Beginner will be confused as heck here. Explain what is quantification, illustrate explicit vs implicit.

Reply:
Yes, I agree. And also what is quanitifier. Will provide examples soon.